nautobot/extras/models/jobs.py

```diff
@@ -342,7 +342,7 @@ def class_path(self):
     @property
     def latest_result(self):
         if self._latest_result is None:
-            self._latest_result = self.results.first()
+            self._latest_result = self.results.only("status").first()
```

Comment:
Would you mind adding a docstring for `latest_result` documenting this restriction/requirement?